geckomat/get_enzyme_data/loadDatabases.m

```diff
@@ -0,0 +1,148 @@
+function databases = loadDatabases(modelAdapter,selectDatabase)
+% loadDatabases
+%   Load or downloads (if necessary) the organism-specific KEGG and UniProt
+%   databases that are required to extract protein information.
+%
+% Input:
+%   modelAdapter    a GECKO organism-specific modelAdapter function that
+%                   contains the taxonomic ID (taxonID, for UniProt) and
+%                   organism code (keggID, for KEGG)
+%   selectDatabase  which databases should be loaded, either 'uniprot',
+%                   'kegg' or 'both'
+%
+% Output:
+%   databases       contains .uniprot and .kegg structures, dependent on
+%                   which databases were selected.
+%
+% Usage: databases = loadDatabases(modelAdapter,selectDatabase)
+
+if nargin<2
+    selectDatabase = 'both';
+end
+geckoPath = findGECKOroot();
```

Comment:
I realize this code is just copied from my previous code, but I think it would be better to move the retrieval of this path to the base model, so it can be overridden. This way, we could create a small test taxon placed in the test folder when we do test cases, and a TestModelAdapter would then point to that folder instead of the standard one. That also makes it possible to provide a custom such database if desired.